kpi/constants.py

```diff
@@ -70,3 +70,19 @@
 
 # KC INTERNAL
 PERM_FROM_KC_ONLY = 'from_kc_only'
+
+# Terms that can be used to search and filter return values
+# from a query `q`
+ASSET_SEARCH_DEFAULT_FIELD_LOOKUPS = [
```

Comment:
I would get rid of `ASSET_SEARCH_DEFAULT_FIELD_LOOKUPS` and `EXPORT_TASK_SEARCH_DEFAULT_FIELD_LOOKUPS` and I would assign their content to `self.default_field_lookups` in each viewset.  

I kept it like this because of the tests.